### DISCUSSION TITLE
3 small Questions or Ideas

### DISCUSSION THREAD

**Discussion by Felinora** [bookfere/Ebook-Translator-Calibre-Plugin#171] [Q&A]
1. Thank you for your nice plugin like Nam Bui in Mobileread asked https://www.mobileread.com/forums/showpost.php?p=4363067&postcount=89  do I try to style the original and the translated text different to easier read through a book.

How do I adress only the original paragraphes and then the translated ones? Do they get an identifer? Like language or so?
I am not trained in CSS so maybe there is a solution but I do not know how to excess it. 
I wanted the original text to be italic or underlined or something else, to easier see that it is one language. Because of the easier readable from black on white and the oppesite I dont want to color the translated text.

2. Another help would be if you could add a line or --------- .............. after every translated and original content to seperate it from the next. Like:

Hallo ich bin ein Text!

Hello I am a text!

.............

Ich bin noch ein Text.

I am another text.
..........

3. I also notice that the metadata Information from the original ebook entry in calibre is not copied in the new entry from the book. Also it do not copy my title. So I need to copy all the metadata into the translated entry. That is a lot of work that could easily skippt. Maybe I have there also a malfunction because I checked "set Target language to metadata" but if i translate a book from English to German in the metadata of the translated one is still "English" in the language column.

Thank you for your hard work with this plugin. It helped me to read more difficult English ebooks in my motherlanguage German.

**Comment by bookfere** [comment]
Thank you for your feedback.

We've noted your first two requirements, and we're in the process of adding features to meet them.

Regarding the "metadata" issue, we've identified a bug that removes all "dc:subject" elements from the metadata. However, "dc:title" remains as expected. Could you please double-check if the title was removed?

As for the "set Target language to metadata" feature, it is working as expected in our testing. Can we confirm whether you clicked the [Save] button to save the settings after checking that option?

We're pleased to provide you with the convenience of reading materials across languages.
